com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -821,32 +1135,36 @@ public bool StartServer()
 
             Initialize(true);
 
-            // If we failed to start then shutdown and notify user that the transport failed to start
-            if (NetworkConfig.NetworkTransport.StartServer())
-            {
-                IsServer = true;
-                IsClient = false;
-                IsListening = true;
-
-                SpawnManager.ServerSpawnSceneObjectsOnStartSweep();
+            IsServer = true;
+            IsClient = false;
 
-                OnServerStarted?.Invoke();
-                return true;
-            }
-            else
-            {
-                Debug.LogError($"Server is shutting down due to network transport start failure of {NetworkConfig.NetworkTransport.GetType().Name}!");
-                Shutdown();
-            }
+            State = NetworkManagerState.LoadingAssets;
 
-            return false;
+            return ResolveNetworkPrefabs();
```

Comment:
Again, per my earlier comment, embedding the transport start calls in ResolveNetworkPrefabs makes it harder to reason about / maintain this.

Reply:
Have a new proposal that will help decouple these, but will probably be a separate PR that builds on this one.